common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -327,6 +327,11 @@ private static byte[] digest(MessageDigest digest, URL url) {
     }
 
     static void tryPatchShadedLibraryIdAndSign(File libraryFile, String originalName) {
+        if (!new File("/Library/Developer/CommandLineTools").exists()) {
+            logger.debug("Can't patch shaded library id as CommandLineTools are not installed." +
+                    " Consider installing CommandLineTools with 'xcode-select --install'");
+            return;
```

Comment:
I wonder if this should be a `warn`, and then not break early in case the tools are installed in  a different location.

Reply:
Maybe we could also call ` xcode-select -p` to check where the tools are located ?